lib/jekyll/commands/doctor.rb

```diff
@@ -67,15 +67,13 @@ def deprecated_relative_permalinks(site)
 
         def conflicting_urls(site)
           conflicting_urls = false
-          urls = {}
-          urls = collect_urls(urls, site.pages, site.dest)
-          urls = collect_urls(urls, site.posts.docs, site.dest)
-          urls.each do |url, paths|
+          url_source_map(site).each do |url, paths|
             next unless paths.size > 1
 
             conflicting_urls = true
-            Jekyll.logger.warn "Conflict:", "The URL '#{url}' is the destination" \
-              " for the following pages: #{paths.join(", ")}"
+            Jekyll.logger.warn "URL Conflict:",
```

Comment:
UX: we mention a `URL` conflict and then display path in the `/tmp` folder? This is confusing. 
Can we improve the output here?

```
Conflict detected.
The following files point to the same URL "/puppies":
  - puppies.html
  - _posts/puppies.md
```

Reply:
`jekyll/tmp/jekyll/` in the sample (opening post) is my source directory.
`jekyll/tmp/jekyll/_site/` being the destination directory.